feat: Add ADK_MCP multi-agent exmaple (#1978)
GoogleCloudPlatform/generative-ai: gemini/mcp/adk_multiagent_mcp_app/static/user_guide.md

# ADK Multi-Agent MCP Client Application

## 1. Introduction

This chatbot demonstrates the use of the ADK Multi-Agent integrated with MCP Clients

Architecture:

![architecture](adk_multiagent.png)

It has three agents: root agent, cocktail agent, and booking agent.

It interacts with the following MCP servers:

> - 1.Cocktail MCP server (Local code)
> - 2.Weather MCP server (Local code)
> 
> - 3.Public Airbnb MCP server (Fetch code via Pypi)

Reviewer: ```suggestion
1. Cocktail MCP server (Local code)
1. Weather MCP server (Local code)
1. Public Airbnb MCP server (Fetch code via PyPi)
```

The Cocktail MCP server has 5 tools:

> - 1.search cocktail by name
> - 2.list all cocktail by first letter
> - 3.search ingredient by name.
> - 4.list random cocktails
> - 5.lookup full cocktail details by ID

Reviewer: ```suggestion
1. search cocktail by name
1. list all cocktail by first letter
1. search ingredient by name.
1. list random cocktails
1. lookup full cocktail details by ID
```

The Weather MCP server has 3 tools:

> - 1.get weather forecast by city name
> - 2.get weather forecast by coordinates
> - 3.get weather alert by state code

Reviewer: ```suggestion
1. get weather forecast by city name
1. get weather forecast by coordinates
1. get weather alert by state code
```

The Airbnb MCP server has 2 tools:

> - 1.search for Airbnb listings
> - 2.get detailed information about a specific Airbnb listing

Reviewer: ```suggestion
1. search for Airbnb listings
1. get detailed information about a specific Airbnb listing
```

## 2. Example questions

Here are some example questions you can ask the chatbot:

> - 'Please get cocktail margarita ID and then full detail of cocktail margarita'
> - 'Please list a random cocktail'
> - 'Please get weather forecast for New York'
> - 'Please get weather forecast for 40.7128,-74.0060'
> - 'I would like to book an Airbnb condo in LA, CA for 2 nights. 04/28 - 04/30, 2025, two adults, no kid'

Reviewer: ```suggestion
- `Please get cocktail margarita ID and then full detail of cocktail margarita`
- `Please list a random cocktail`
- `Please get weather forecast for New York`
- `Please get weather forecast for 40.7128,-74.0060`
- `I would like to book an Airbnb condo in LA, CA for 2 nights. 04/28 - 04/30, 2025, two adults, no kid`
```
